templates/browse.html

```diff
@@ -40,36 +44,120 @@
 </nav>
 <!--End of Navbar-->
 <!--Card Section-->
-<div class="container container-fluid mt-4">
-  <div class="card" style="width: 18rem">
-    <img src="https://fakeimg.pl/200x200" class="card-img-top" alt="..." />
-    <div class="card-body">
-      <h5 class="card-title">Card title</h5>
-      <p class="card-text">
-        Some quick example text to build on the card title and make up the bulk
-        of the card's content.
-      </p>
-    </div>
-    <!--Buttons-->
-    <div class="card-body">
-      <a href="#" class="card-link">Share</a>
-      <a href="#" class="card-link">Download</a>
-    </div>
-  </div>
-  <div class="card mt-2" style="width: 18rem">
-    <img src="https://fakeimg.pl/200x200" class="card-img-top" alt="..." />
-    <div class="card-body">
-      <h5 class="card-title">Card title</h5>
-      <p class="card-text">
-        Some quick example text to build on the card title and make up the bulk
-        of the card's content.
-      </p>
-    </div>
-    <!--Buttons-->
-    <div class="card-body">
-      <a href="#" class="card-link">Share</a>
-      <a href="#" class="card-link">Download</a>
-    </div>
+<div class="container container-fluid embed-responsive embed-responsive-16by9 mt-3">
```

Comment:
Visually looks really good but none of this uses the backend routes I created. It should get the posts already created and display them. Also you are missing the post details page but that ticket has been moved to finished.